app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from datetime import datetime
+from time import sleep
+
+
+def file_every_second():
+    while True:
+        time = datetime.now()
```

Comment:
You can put datetime.now() in your write function. This way you won't have to create a new variable on every iteration